doc/about.rst

```diff
@@ -22,11 +22,21 @@ Governance
 The decision making process and governance structure of scikit-learn is laid
 out in the :ref:`governance document <governance>`.
 
-Authors
--------
+.. _authors:
+
+The people behind scikit-learn
+-------------------------------
+
+Scikit-learn is a community project, developed by a large group of
+people, all across the world. A few teams, listed below, have central
+roles, however a complete list of contributors can be found `on github
+<https://github.com/scikit-learn/scikit-learn/graphs/contributors>`__.
+
+Core Developer Team
+....................
 
-The following people are currently core contributors to scikit-learn's development
-and maintenance:
+The following people are currently core developers, in charge of
```

Comment:
ok now I'm confused, I thought contributor was referring to all kind of contributions, hence the different teams which to me all are contributors. This section is only about the core devs. I'm happy with Maintainers Team as well (this is what I proposed in yesterday meeting but it was ruled out quickly).

Reply:
Yes, this is also what I had in mind: this subsection is not about comms, dev relations, etc...

Reply:
The authors list includes maintainers only, so this line should be:

```suggestion
The following people are currently maintainers, in charge of
```

Reply:
You are right that it is called the "maintainers" group in the governance document

By the way, I find that the governance document is hard to read here because the section titles don't convey well the nesting:
https://scikit-learn.org/stable/governance.html#governance

Should I try to change, for instance using a listed description rather than section titles?

Reply:
I agree that the nesting is not clear

Reply:
OK, I'm changing a bit the layout